osu.Game/Overlays/DialogOverlay.cs

```diff
@@ -88,16 +94,14 @@ protected override void PopIn()
         protected override void PopOut()
         {
             base.PopOut();
+            this.FadeOut(PopupDialog.EXIT_DURATION, Easing.InSine);
 
-            lowPassFilter.CutoffTo(AudioFilter.MAX_LOWPASS_CUTOFF, 100, Easing.InCubic);
+            // PopOut is called as part of VisibilityContainer's initialisation logic, but we don't want it to interact with a potentially waiting dialog.
+            if (!IsLoaded) return;
```

Comment:
This change should be irrelevant now that the push is delayed until the dialog is loaded, yeah?